### PR TITLE
Change arrow head fill property to black

### PR DESCRIPTION
**Pull request by ibdafna**
Signed-off-by: Itay Dafna <i.b.dafna@gmail.com>

### CODE REVIEW

**Comment by martinRenou** [comment]
I guess we need to regenerate the visual regression tests references

**Comment by ibdafna** [comment]
Yeah, also needs a bit it refactoring with the CSS class names for
dark/white themes. Will sort this week!

File: js/less/bqplot.less [hunk 1]
```diff
@@ -63,7 +63,7 @@
   --bq-pie-polyline-stroke: var(--bq-content-font-color);
 
   --bq-network-node-element-fill: #666;
-  --bq-linkarrow-fill: #ccc;
+  --bq-linkarrow-fill: rgb(0, 0, 0);
```
Comment (martinRenou) [inline js/less/bqplot.less:66]: I guess you want to use a CSS variable for that?

Comment (ibdafna) [reply js/less/bqplot.less]: Yes, absolutely 😄

**Comment by martinRenou** [comment]
@ibdafna is it ready to be merged? :)

**Pull request merged**